docs/_layout/sidebar.md

```diff
@@ -1,23 +1,23 @@
 <!-- markdownlint-disable-next-line -->
 - Discord
-  - [Rules](/discord/discord-readme.md)
-  - [Moderation Guide](/moderators/discord.md)
-  - [Self-Roles](/discord/roles.md)
-  - [Welcome DM](/mee6/welcome-dm.md)
-  - [Allowed Links](/nightbot/allowed-links.md)
+  - [Rules](../discord/discord-readme.md)
+  - [Moderation Guide](../moderators/discord.md)
+  - [Self-Roles](../discord/roles.md)
+  - [Welcome DM](../mee6/welcome-dm.md)
+  - [Allowed Links](../nightbot/allowed-links.md)
 
 - GitHub
-  - [GitHub Maintainer Guide](/moderators/github.md)
+  - [GitHub Maintainer Guide](../moderators/github.md)
 
 - Tips
-  - [Finding Open Source Projects](/tips/finding-open-source-projects.md)
-  - [Git Tips and Tricks](/tips/git.md)
-  - [How to Use GitHub](/tips/githubtips.md)
-  - [Common Terms](/tips/glossary.md)
-  - [Open Source Resources](/tips/OpenSourceResources.md)
-  - [Open Source Tips](/tips/OpenSourceTips.md)
-  - [Glossary](/tips/glossary.md)
+  - [Finding Open Source Projects](../tips/finding-open-source-projects.md)
+  - [Git Tips and Tricks](../tips/git.md)
+  - [How to Use GitHub](../tips/githubtips.md)
+  - [Common Terms](../tips/glossary.md)
+  - [Open Source Resources](../tips/OpenSourceResources.md)
+  - [Open Source Tips](../tips/OpenSourceTips.md)
+  - [Glossary](../tips/glossary.md)
 
 - Templates
-  - [YouTube Generic Template](/youtube/generic.md)
-  - [LiveStream Review Template](/youtube/livestream.md)
```

Comment:
I don't think these paths need changing?

Reply:
The md files are outside the _layout folder. If it is not being used in the website then it should not be changed.

Reply:
The files are outside the folder but when the app is built and deployed, I think it is all working fine https://eddiehubcommunity.github.io/support/#/discord/discord-readme

Reply:
Thanks for pointing that out! I have removed my changes. Other than that, I have removed the link of the livestream.md file.